packages/discovery-react-components/src/components/DiscoverySearch/DiscoverySearch.tsx

```diff
@@ -122,7 +122,11 @@ export interface SearchApiIFC {
   performSearch: (searchParameters: DiscoveryV2.QueryParams, resetAggregations?: boolean) => void;
   fetchAutocompletions: (nlq: string) => Promise<void>;
   fetchAggregations: (searchParameters: DiscoveryV2.QueryParams) => Promise<void>;
-  fetchDocuments: (filterString: string, searchResponse: DiscoveryV2.QueryResponse | null) => void;
+  fetchDocuments: (
+    filterString: string,
+    collections?: string[],
+    searchResponse?: DiscoveryV2.QueryResponse
+  ) => void;
```

Comment:
This is a BREAKING CHANGE: we need to specify a collection ID as well, when fetching documents based on ID, since doc IDs are only unique within collections, not projects.